### DISCUSSION TITLE
reduce the height of tabs

### DISCUSSION THREAD

**Discussion by shalva97** [minbrowser/min#1739] [Ideas]
I think they look tall, just like on Firefox. Edge and Opera's tabs  are shorter and it looks nice.

**Comment by PalmerAL** [comment]
They're approximately the same height as Firefox's compact mode, and shorter than Firefox in regular mode, although I think they look taller because they fill the whole navbar:
<img width="696" src="https://user-images.githubusercontent.com/10314059/133872905-29138ebf-18b2-46fb-a498-c9a11b0087b0.png">

On Windows, it looks worse because we have a drag area above the tabs, but I haven't seen a good solution to that so far.

We could do like a "compact mode" as well. My guess is that the font sizes and buttons would end up sort of uncomfortably small, so it would need to be an option, but for people who mostly use the keyboard anyway it might be nice.
